tests/test_admin_views.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+
+from django.http import Http404
+
+from accession import admin_views
+
+from .factories import DonorFactory, AccessionFactory
+
+pytestmark = pytest.mark.django_db
+
+
+class TestDuplicates():
+
+    def test_returns_200_with_mapped_model(self, rf, admin_user):
```

Comment:
What do you mean by _mapped_? I see there is a `field_mapping` but this is unrelated to the `Donor` model.